resource_tracker/models.py

```diff
@@ -45,7 +46,7 @@ def raise_if_attribute_name_exist(self, name):
 
     def init_attribute(self, attribute):
         for resource in self.resources.all():
-            resource.set_attribute(attribute, 0)
+            resource.attributes.get_or_create(attribute_type=attribute, defaults={'value': 0})
```

Comment:
It will create the attribute on resource with value 0 if it does not exist

Reply:
set_attribute doing the same (use set_attribute instead of get_or_create in all others methods?)

Reply:
Set attribute put 0 everywhere, even if the resource already exists and already has a value.
Get_or_create try to get the resource attribute and if it doesn't exist, it will create it with data in "defaults" here: value = 0

Reply:
yes but I think is better to change in set_attribute method (still using set_attribute here)

Reply:
And change method bellow to : 
```python
def set_attribute(self, attribute_type, value=0):
    resource.attributes.get_or_create(attribute_type=attribute, defaults={'value': value})
```